src/test/resources/eureka-server/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM netflixoss/tomcat:7.0.64
```

Comment:
I would also support creating and publishing our own Docker container like you mentioned in the PR overview. What pros & cons does that have?

Reply:
See my comment on the war related conversation